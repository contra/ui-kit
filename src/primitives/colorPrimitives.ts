/* eslint-disable */
/* Generated file. Do not update manually! */

export const colorPrimitives = {
  alertError: '#e06050',
  alertInformational: '#5b9be0',
  alertSuccess: '#66ad5a',
  alertWarning: '#e5973c',
  brandBlack: '#131313',
  brandBlue: '#6ca3e0',
  brandGreen: '#d2d943',
  brandLightGray: '#f6f6f6',
  brandOrange: '#ff9e31',
  brandPurple: '#8f6fdb',
  brandSalmon: '#f2a28a',
  brandWatermelon: '#ff5a5e',
  brandWhite: '#ffffff',
  brandYellow: '#f2c94c',
  gray10: '#f6f6f6',
  gray20: '#eeeeee',
  gray30: '#e3e3e3',
  gray40: '#d1d1d1',
  gray50: '#acacac',
  gray60: '#8b8b8b',
  gray70: '#646464',
  gray80: '#515151',
  gray90: '#333333',
  green10: '#f0f7ee',
  green20: '#e0efde',
  green30: '#d1e6ce',
  green40: '#c2debd',
  green50: '#b3d6ad',
  green60: '#a3ce9c',
  green70: '#94c68c',
  overlay10: 'rgba(19,19,19,0.5)',
  overlay20: 'rgba(0,0,0,0.77)',
  red10: '#fcefed',
  red20: '#f9dfdc',
  red30: '#f6cfcb',
  red40: '#f3bfb9',
  red50: '#f0b0a7',
  red60: '#eca096',
  red70: '#e99084',
  socialInvision: '#dc395f',
  socialReact: '#53c1de',
  socialTwitter: '#4aa1eb',
  uiBlackDisabled: 'rgba(19,19,19,0.4)',
  uiBlackHighEmphasis: '#131313',
  uiBlackMediumEmphasis: 'rgba(19,19,19,0.67)',
  uiErrorFocus: '#ce0e19',
  uiErrorRegular: '#e90f1c',
  uiFilterGreen: '#cdedc7',
  uiInputBackground: 'rgba(242,242,242,0.4)',
  uiSelected: '#dfeaf9',
  white10: 'rgba(255,255,255,0.36)',
  white20: 'rgba(255,255,255,0.43)',
  white30: 'rgba(255,255,255,0.63)',
  white40: 'rgba(255,255,255,0.86)',
  yellow10: '#fdf6e0',
  yellow20: '#fae7b1',
  yellow30: '#f6d77f',
  yellow40: '#f2c94c',
  yellow50: '#f0bc27',
  yellow60: '#edb104',
  yellow70: '#eda400',
};
