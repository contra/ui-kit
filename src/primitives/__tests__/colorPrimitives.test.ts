/**
 * This snapshot test is here to make sure we are intentional
 * about any updates to this file.
 *
 * 1. Did the format change? If so did we fix how that effects things down stream?
 * 2. Color value changes or removal need treated like a BREAKING change.
 */

import { colorPrimitives } from '../colorPrimitives';

describe('colorPrimitives', () => {
  it('should match snapshot', () => {
    expect(colorPrimitives).toMatchSnapshot();
  });
});
