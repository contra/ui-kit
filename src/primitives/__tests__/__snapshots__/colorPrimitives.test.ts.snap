// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colorPrimitives should match snapshot 1`] = `
Object {
  "alertError": "#e06050",
  "alertInformational": "#5b9be0",
  "alertSuccess": "#66ad5a",
  "alertWarning": "#e5973c",
  "brandBlack": "#131313",
  "brandBlue": "#6ca3e0",
  "brandGreen": "#d2d943",
  "brandLightGray": "#f7f7f7",
  "brandOrange": "#ff9e31",
  "brandPurple": "#8f6fdb",
  "brandSalmon": "#f2a28a",
  "brandWatermelon": "#ff5a5e",
  "brandWhite": "#ffffff",
  "brandYellow": "#f2c94c",
  "gray10": "#f7f7f7",
  "gray20": "#eeeeee",
  "gray30": "#e3e3e3",
  "gray40": "#d1d1d1",
  "gray50": "#acacac",
  "gray60": "#8b8b8b",
  "gray70": "#646464",
  "gray80": "#515151",
  "gray90": "#333333",
  "socialInvision": "#dc395f",
  "socialReact": "#53c1de",
  "socialTwitter": "#4aa1eb",
  "uiBlackDisabled": "rgba(19,19,19,0.4)",
  "uiBlackHighEmphasis": "#131313",
  "uiBlackMediumEmphasis": "rgba(19,19,19,0.67)",
  "uiErrorFocus": "#ce0e19",
  "uiErrorRegular": "#e90f1c",
  "uiFilterGreen": "#cdedc7",
  "uiInputBackground": "rgba(242,242,242,0.4)",
  "uiSelected": "#dfeaf9",
  "yellow10": "#fdf6e0",
  "yellow20": "#fae7b1",
  "yellow30": "#f6d77f",
  "yellow40": "#f2c94c",
  "yellow50": "#f0bc27",
  "yellow60": "#edb104",
  "yellow70": "#eda400",
}
`;
